### PR TITLE
Feature/event calendar

### PR DESCRIPTION
**Pull request by StepanPotiienko**

### CODE REVIEW

File: app/(client)/event-calendar/page.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+              {isDesc ? (
+                <SortAsc className="scale-150" />
+              ) : (
+                <SortDesc className="scale-150" />
+              )}
+            </Button>
+          </div>
+        </div>
+        <div className="w-full flex items-center justify-center max-[1280px]:px-3">
+          <div className="grid grid-cols-5 gap-5 gap-y-10 w-full max-[1280px]:flex max-[1280px]:flex-col max-[1280px]:gap-y-3">
```
Comment (Kyryloloshka) [inline app/(client)/event-calendar/page.tsx:108]: too soon the grid turns into a vertical list, at intermediate widths you can make a grid of 2-4 elements in a row

File: app/(client)/event-calendar/page.tsx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            />
+            <Event
+              title={'День перваша'}
+              description={
+                'Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис '
+              }
+              location={'КПІ'}
+              tag={'перваш'}
+              date={'31.09.2024'}
+              time={'15:00-16:00'}
```
Comment (Kyryloloshka) [inline app/(client)/event-calendar/page.tsx:167]: all these elements <Event/> are the same, only the filling is different. All data must be taken from the backend, and there must be a map through all the data

File: app/(client)/event-calendar/page.tsx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        autoComplete="true"
+        value={value}
+        onValueChange={newValue => setValue(newValue)}
+        onClick={() => {
+          setHasSetValue(false);
+        }}
+      />
+      <CommandList className={hasSetValue ? 'hidden w-0 h-0' : 'block'}>
+        <CommandEmpty>Не вдалось знайти захід.</CommandEmpty>
+        {EventsList.map(i => (
```
Comment (Kyryloloshka) [inline app/(client)/event-calendar/page.tsx:49]: Searching must be through real data

**Review by Kyryloloshka — CHANGES_REQUESTED** [review]

File: app/(client)/event-calendar/page.tsx (hunk 3 above)
Comment (StepanPotiienko) [reply app/(client)/event-calendar/page.tsx:49]: There is no real data yet. EventsList contains the data that should be gathered from API.

File: app/(client)/event-calendar/page.tsx (hunk 2 above)
Comment (StepanPotiienko) [reply app/(client)/event-calendar/page.tsx:167]: I am told to add real data after the pr is merged. So there is no real data yet.

**Review by Markusplay — APPROVED** [review]

**Pull request merged**